tests/integration/browser.worker.js

```diff
@@ -24,18 +24,25 @@ if (typeof window.Worker === 'function' &&
 
 function runTests() {
 
+  var worker;
+
+  before(function () {
+    worker = new Worker('worker.js');
+    worker.postMessage(['source', sourceFile]);
+  });
+
+  after(function () {
+    worker.terminate();
+  });
```

Comment:
creating and terminating a new worker for every test here was really not necessary